editor/modes/visual-editor/block.js

```diff
@@ -137,15 +137,13 @@ class VisualEditorBlock extends wp.element.Component {
 			wrapperProps = settings.getEditWrapperProps( block.attributes );
 		}
 
-		// Disable reason: Each block can receive focus but must be able to contain
-		// block children. Tab keyboard navigation enabled by tabIndex assignment.
+		// Disable reason: Each block can be selected by clicking on it
```

Comment:
Should the comment be reverted back to its previous text? Or at least updated to reflect that it accepts focus? Seems like suggesting "clicking on it" as reason for disabling an accessibility rule is destined to... provoke response 😄